tests/gpflow/kernels/test_broadcasting.py

```diff
@@ -44,12 +44,19 @@
     # Following kernels do not broadcast: see https://github.com/GPflow/GPflow/issues/1339
     pytest.param(kernels.ArcCosine, marks=pytest.mark.xfail),  # broadcasting not implemented
     pytest.param(kernels.Coregion, marks=pytest.mark.xfail),  # broadcasting not implemented
-    pytest.param(kernels.Periodic, marks=pytest.mark.xfail),  # broadcasting not implemented
     pytest.param(kernels.ChangePoints, marks=pytest.mark.xfail),  # broadcasting not implemented
     pytest.param(kernels.Convolutional, marks=pytest.mark.xfail),  # broadcasting not implemented
 ]
 
 
+def broadcast_no_no_active_dims(kernel):
```

Comment:
```suggestion
def check_broadcasting(kernel):
```
might make it clearer what's happening?